### ISSUE TITLE
[a11y]: color contrast is low and missing labels for the for form 

### ISSUE THREAD

**Issue by CBID2** [DevLeonardoCommunity/github-stats#103]
### Describe your issue

I ran some accessibility tests on the website and here are the results.

### Accessibility Insights

- Element has insufficient color contrast of 2.93 (foreground color: #e1e6fe, background color: #3c83f6, font size: 10.5pt (14px), font weight: normal). Contrast ratio: 2.93:1
![contrast checker results](https://github.com/DevLeonardoCommunity/github-stats/assets/105683440/afb9776c-c934-4018-918f-07aaf6bf109d)

- Expected contrast ratio of 4.5:1 according to [WCAG 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).

### WAVE Tester
![wave-accessibility-results](https://github.com/DevLeonardoCommunity/github-stats/assets/105683440/87e25317-5d01-41dc-87c6-263996cd9780)

### Steps to reproduce

Not applicable

### Would you like to work on this issue?

Yes

**Comment by Balastrong** [comment]
Thanks for doing the tests! I'll be really happy to see a PR for this :D
